draft-linker-diem-requirements.md

```diff
@@ -55,6 +55,45 @@ TODO Introduction
 
 ## Requirements
 
+### Authenticity
+
+Validators MUST be able to verify that an emblem was issued for the respective bearer, issued by the claimed issuer (where applicable), and that none of its associated data was changed.
+
+Use cases that have this requirement:
+
+### Issuance and Authorization
+
+Validators MUST be able to verify that an emblem issuer was authorized to issue the emblem.
+Authorizing parties MAY limit what emblems an issuer can issue, e.g., they MAY limit issuance to certain bearers.
+
+Use cases that have this requirement:
+
+#### Decentralization
+
+Anyone MUST be able to authorize an issuer.
+
+Use cases that have this requirement:
+
+#### Accountability
+
+Emblem issuance and authorization of issuers MUST be attributable to issuers and authorizing parties.
+Authorizing parties and emblem issuers MUST NOT be able to repudiate that they issued an emblem or authorization.
```

Comment:
Could you provide a practical example of this requirement?

Reply:
I hope the first part of the requirement is clear, namely, that emblems and authorizations must carry the identity of the signer. This is very typical for certificate-style designs, but not a given.

The second part is about being able to hold illicit display of digital emblems accountable. If someone illegitamtely displays an emblem (which, for the most part, can only be determined after the fact), we must be able to hold that display accountable. Using digital signatures already goes a long way to realize this requirement!

One could make the requirement stronger and require accountability directly, though, it may be unclear what this means.

Reply:
Okay, that makes sense - thanks!